_posts/2021-06-25-app.md

```diff
@@ -0,0 +1,104 @@
+description: "Entwicklung einer multi-plattform Navigations-Applikation"
+---
+
+Dieser Artikel soll Einblicke geben in die Idee und Entwicklung der ersten mobilen Lastenrad-Routing Applikation. Um mehr über den zugrundeliegenden Algorithmus zu erfahren lohnt sich ein Blick in unseren Artikel ["Der technische Aufbau unseres Lastenrad Routing"](https://cargorocket.de/2021/05/31/routing.html)
+Zum Ende des Artikels findet sich eine technische Beschreibung des Quellcodes der App. 
+
+## Motivation
+
+### Mobidata Hack
+Schon während des MobiData BW 2020 Hackathons, bei dem die Idee zum CargoRocket Lastenradrouting entstand, überlegten wir, wie wir das fertige Routing am besten präsentieren, validieren und am Ende auch verwendbar machen könnten. Schnell war klar, dass sich eine App für mobile Endgeräte sich besonders anbietet. Leider kam es während dem Hackathon zu [Technischen Schwierigkeiten](###Startschwierigkeiten), die uns daran hinderten schon zum Ende der 48h eine funktionsfähige App zu präsentieren. Als Ergebnis reichten wir als Routing-Demonstator eine Web-Applikation ein, die das Basisrouting auf eine einfache Weise visualisierte ([Github Repo [Archived]](https://github.com/CargoRocket/Launchpad)).
+
+<figure>
+  <img alt="Screenshot of Launchpad Web-Applikation" src="/assets/images/launchpad-1.0.png" style="max-width: 100%;" />
+  <figcaption>Screenshot der Launchpad Web-Applikation</figcaption>
+</figure> 
+
+### Der nächste Schritt
+Für den folgenden Förderzeitraum steckten wir uns ambitionierte Ziele. Innerhalb von 3 Monaten sollte eine mobile Applikation für Android und iOS entwickelt werden.
+
+Aktuell (20.06.21) befindet sich die Android version im BETA-Test.
+
+Während dieser Testphase erhoffen wir uns Feedback zur App, aber auch zu unserem Routing und unterliegenden Index.
+Direkt Tester:Innen ein Produkt zum Ausprobieren zur Verfügung zu stellen ist aus unserer Sicht eine riesige Chance hier schnell und effizient breite Rückmeldungen einholen zu können.
+Tester:Innen haben außerdem die Möglichkeit sich direkt auf der geplanten route navigieren zu lassen und vorort Hindernisse zu melden.
+
+Die iOS version der App wird im Anschluss des BETA-Tests unter Android in einigen Wochen im App-Store zum Download zur Verfügung stehen.
+
+### Datenerhebung
+
+Zur späteren Analyse zeichnet die App die Routen der Tester auf und wird diese später anonymisiert an uns übertragen. Dabei ist uns besonders wichtig, dass aus diesem Datensatz der tatsächliche Wohnort / Arbeitsstädte unserer Tester:Innen nicht hervorgeht. Hierzu wird vor Beginn der Datenübertagung hier dazu ein Artikel erscheinen. Aus diesem Datensatz lässt sich die tatsächliche Reisezeit und umfahrene Hindernisse entnehmen.
+
+Zum Einstieg in die App fragen wir auch die Fahrzeugdimensionen und Art des Fahrzeuges ab. Diese Daten lassen wir auch in das Routing einfließen, sodass beispielsweise Poller mit zu niedriger Breite umfahren werden können.
```

Comment:
sollen wir hier erwhähnen, dass das so geplant aber noch nicht implementiert ist?

Reply:
Idk. Würde das schon so aufnehmen. Auf App und middlewareseite ist dafür schon alles da. @ohrie wann schätzt du können wir die Info in unser Routing einfließen lassen?

Reply:
Gute Frage. Ich würde jetzt einfach mal als Deadline (vor allem für mich) Ende Juli/Anfang August  sagen :)